ontotext-yasgui-web-component/src/plugins/yasr/pivot-table-plugin.ts

```diff
@@ -21,20 +28,25 @@ export class PivotTablePlugin implements YasrPlugin {
     }
   }
 
-  canHandleResults(): boolean {
-    // TODO check is this correct
-    return !!this.yasr.results && this.yasr.results.getVariables() && this.yasr.results.getVariables().length > 0;
+  initialize(): Promise<void> {
+    return new Promise(resolve => {
+      HtmlUtil.addScriptTag('https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js');
+      HtmlUtil.addScriptTag('https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.2/jquery.min.js');
+      HtmlUtil.addScriptTag('https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js');
+      HtmlUtil.addScriptTag('https://www.google.com/jsapi');
+      HtmlUtil.addScriptTag('https://www.gstatic.com/charts/loader.js');
+      HtmlUtil.addScriptTag('https://pivottable.js.org/dist/pivot.js');
+      HtmlUtil.addScriptTag('https://pivottable.js.org/dist/d3_renderers.js');
+      HtmlUtil.addScriptTag('https://pivottable.js.org/dist/gchart_renderers.js', () => resolve());
```

Comment:
Not sure this is quite correct. Why you resolve only here

Reply:
Because all scripts are loaded, this is the last one.

Reply:
I guess you rely on the fact that you set scripts to be loaded synchronously which although is not quite desireable and we should probably consider some approach for asyncronous scrips loading and still have proper synchronization between loading and the rendering

Reply:
Also, same scripts and styles are appended again and again during window resizing or switching between editor tabs

Reply:
I'm thinking about this. I could put all the scripts in some src folder and load them from there, but we can discuss it.